config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_accesscontextmanagerserviceperimeterresources.accesscontextmanager.cnrm.cloud.google.com.yaml

```diff
@@ -86,15 +221,37 @@ spec:
                     description: 'Namespace of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/'
                     type: string
                 type: object
-              resource:
-                description: |-
-                  Immutable. A GCP resource that is inside of the service perimeter.
-                  Currently only projects are allowed.
-                  Format: projects/{project_number}.
-                type: string
+              projectRef:
```

Comment:
Is projectRef immutable?  I worry projects members might "leak" if is mutable & we change it.

Reply:
It is immutable in TF so it is immutable in KCC: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/services/accesscontextmanager/resource_access_context_manager_service_perimeter_resource.go#L58
Note that this is not a hierarchical reference and I suggested a renaming in servicemapping.

Reply:
I renamed it to "resourceRef" to avoid confusion.